lib/jsonapi_compliable/sideload/polymorphic_belongs_to.rb

```diff
@@ -0,0 +1,86 @@
+class JsonapiCompliable::Sideload::PolymorphicBelongsTo < JsonapiCompliable::Sideload::BelongsTo
+  class Group
+    attr_reader :name, :calls
+
+    def initialize(name)
+      @name = name
+      @calls = []
+    end
+
+    def method_missing(name, *args, &blk)
+      @calls << [name, args, blk]
+    end
+  end
+
+  class Grouper
+    attr_reader :column_name
+
+    def initialize(column_name)
```

Comment:
`field_name` is probably more correct, since column specifically implies a db, yes?

Reply:
I think `field_name` is fine. I was consciously making this choice because I wanted it to be expressive, similar to `foreign_key` even when it's not a foreign key. But `field_name` works.